pkg/kubernetes/client.go

```diff
@@ -248,20 +231,59 @@ func BuildClientConfig(masterURL string, kubeConfigPath string, timeout time.Dur
 	return kubeCfg, nil
 }
 
-// CountRunningLoadtests used in metrics to report running loadtests
-func (c *Client) CountRunningLoadtests() int64 {
+// CountExistingLoadtests used in metrics to report running loadtests
+func (c *Client) CountExistingLoadtests() (map[string]int64, map[string]int64, error) {
 	tt, err := c.ltClient.List(context.Background(), metaV1.ListOptions{})
 	if err != nil {
 		c.logger.Error("Couldn't list existing loadtests", zap.Error(err))
-		return 0
+		return nil, nil, err
+	}
+
+	var phaseCount = map[string]int64{
```

Comment:
Use LoadTestPhase here instead of string and the constants instead of the key values.
"creating" -> LoadTestCreating